### DISCUSSION TITLE
Creating IOM OpenModelResult from text file

### DISCUSSION THREAD

**Discussion by Rockambole** [idea-statica/ideastatica-public#432] [Features and Ideas]
Hello all,

From my FEM analysis I got a table of end forces for N load cases and M members. 6 dofs per member end (FX,FY,FZ,MX,MY,MZ).
I’m trying to write code to populate an OpenModelResult object with these data. I already have an OpenModel object with model data (nodes, members, sections etc.).

Looking through the IOM examples, I can’t really find something that covers this case. There are references to Member1D and Element1D, reference id’s etc.
It’s quite confusing.

My idea is to create N load cases, each with 6 load values at M member ends (end 1.0). That’s it.
Is there a boiled down example for a case like this?

Thank you.

Regards, O.S.

**Comment by nathanlukeIDEA** [comment]
Hi @Rockambole,

Have you seen the example code on this page?: https://developer.ideastatica.com/docs/iom/iom_open_model_result.html

The structure is Members > Positions > Load Cases. Apologies if this causes confusion and is a little hard to read in the xml.

Results also depend on the start/end of the member. If all the elements in the node end are in the connection then only providing the force at position 1.0 should work. 

Let me know if this does not help. 

Cheers
Nathan
